tests/provision/hardware/test.sh

```diff
@@ -33,6 +33,17 @@ rlJournalStart
         rlAssertNotGrep "warn -> fail C000 fmf node failed schema validation" $rlRun_LOG
     rlPhaseEnd
 
+    rlPhaseStartTest "Check early exit on data size validation errors"
```

Comment:
Interesting, we do not run these tests by default 😭.
https://github.com/teemtee/tmt/blob/4ca47d4dd8c24e640e249b91ac656b11c8a7e4a7/tests/provision/hardware/main.fmf#L2-L5
Maybe try to drop the `provision-only`